lib/bank/account.rb

```diff
@@ -0,0 +1,37 @@
+module Bank
+  class Account
+    # attr_accessor creates an implicit balance method
+    attr_accessor :id, :balance
+
+    def initialize(id, initial_balance)
+      raise ArgumentError, "Negative balance is invalid." if initial_balance.to_i < 0
+
+      @id = id
+      @balance = initial_balance.to_i
```

Comment:
Something to be careful about is that money is allowed to be fractional, so using integers limits you to whole dollars, so  you may get some funky behavior later.